src/lib/mocha/mocha.js

```diff
@@ -70,7 +70,7 @@ Mocha.prototype.start = function (callback) {
     mochaConfig: JSON.stringify(this.options.mochaConfig)
   });
 
-  self.child = cp.fork(path.join(__dirname, 'mocha-wrapper.js'), _.union(
+  self.child = cp.fork(path.join(__dirname, 'mocha-wrapper-instance.js'), _.union(
```

Comment:
no, that's it. You're right. I wish there was a nice way to do it than to have a file, but this is the implication of dealing with processes I guess.